client/src/Editor.tsx

```diff
@@ -63,9 +63,15 @@ const Editor: React.FC<{setRestart?, onDidChangeContent?, value: string}> =
     const client = new LeanClient(socketUrl, undefined, uri, showRestartMessage)
     const infoProvider = new InfoProvider(client)
     const div: HTMLElement = infoviewRef.current!
-    const infoviewApi = renderInfoview(infoProvider.getApi(), div)
+    const imports = {
```

Comment:
This loads the infoview from a CDN. We could alternatively copy it over to `client/dist`. I am not sure that this matters.

Reply:
I'd prefer to serve the files from our own server because strictly speaking, we otherwise violate the GDPR. Here's an explanation of this (in this case about Google fonts, but the same applies to any non-European server): https://blog.runcloud.io/google-fonts-gdpr/

Reply:
Very good, I set up Webpack to copy things over as assets.